plugins/wpe-headless/wpe-headless.php

```diff
@@ -55,7 +55,7 @@
  */
 function wpe_headless_activated_plugin( $plugin, $network_wide ) {
 	if ( ! defined( 'WP_CLI' ) && ! $network_wide && WPE_HEADLESS_PATH === $plugin ) {
-		wp_safe_redirect( esc_url( admin_url( 'options-general.php?page=wpe-headless-settings' ) ) );
+		wp_safe_redirect( esc_url( admin_url( 'options-general.php?page=wpe-headless-settings&new_activation=1' ) ) );
```

Comment:
We'll need to use [`esc_url_raw`](https://developer.wordpress.org/reference/functions/esc_url_raw/) instead of [`esc_url`](https://developer.wordpress.org/reference/functions/esc_url/), as `esc_url` will convert the "&" to an HTML entity. It looks like `esc_url_raw` is the preferred method as well for redirects.

https://developer.wordpress.org/reference/functions/esc_url_raw/#more-information

Reply:
Nice catch! Change made.